### ISSUE TITLE
Error in Boom instance

### ISSUE THREAD

**Issue by jquiceno** [hapijs/boom#257]
#### Support plan

* *which support plan is this issue covered by?* (e.g. Community, Core, Plus, or Enterprise):  Community
* *is this issue currently blocking your project?* (yes/no): no
* *is this issue affecting a production system?* (yes/no): no

#### Context

* *node version*: v10.16.0
* *module version*: 8.0.1, 
* *environment* (e.g. node, browser, native): Node
* *used with* (e.g. hapi application, another framework, standalone, ...): hapi application
* *any other relevant information*: 

#### How can we help?

The problem is when I try to instantiate a new object in Boom

```js
const Boom = require('@hapi/boom')
const e = new Boom('Test error')
```

This generates an error like this: ```TypeError: Boom is not a constructor```

My implementation is something similar to this:

```js
try {
      message = await message.data()

      if (!message) {
        throw Boom.badRequest('Message not found or invalid')
      }

      return Promise.resolve(message)
    } catch (e) {
      return Promise.reject(new Boom(e))
    }
```

This problem only appears in version 8.xx, when returning to version 7.xx this does not happen, I thought it could be an update but I can see that in the documentation of version 8.xx it still exists, this makes me think What is a problem.

Try the problem directly in npm runkit, and the problem continues.
https://npm.runkit.com/@hapi/boom

**Comment by hueniverse** [comment]
You should read the release notes when upgrading to a new major version. You need to do ‘new Boom.Boom();’

**Comment by AdriVanHoudt** [comment]
@hueniverse did you not release this change in 8.0.1 as a patch? Seems weird to change an interface in a patch version no?

**Comment by hueniverse** [comment]
8.0.0 was unpublished so no, not a patch.

**Comment by AdriVanHoudt** [comment]
But you published 8.0.1 a day after 8.0.0 no? It is safe to say that multiple people have upgraded (including me) during that time and then saw a patch version after (so as an actual patch version not major).

**Comment by hueniverse** [comment]
8.0.0 was a mistake. It wasn't ready.
